packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx

```diff
@@ -51,21 +49,28 @@ export function LocationsView({
   const {
     pageItems,
     hasError,
+    message,
     isPaginatePreviousDisabled,
     isPaginateNextDisabled,
     page,
     isLoading,
+    searchPlaceholder,
```

Comment:
Just noting that the same feedback related to `searchPlaceholder` in `LocationsView` applies here as well